app/src/pages/inside/launchesPage/modals/launchCompareModal/launchCompareModal.jsx

```diff
@@ -0,0 +1,262 @@
+import { Component } from 'react';
+import ReactDOMServer from 'react-dom/server';
+import PropTypes from 'prop-types';
+import C3Chart from 'react-c3js';
+import d3 from 'd3';
+import classNames from 'classnames/bind';
+import { fetch, dateFormat } from 'common/utils';
+import {
+  STATS_AB_TOTAL,
+  STATS_ND_TOTAL,
+  STATS_PB_TOTAL,
+  STATS_SI_TOTAL,
+  STATS_TI_TOTAL,
+  STATS_FAILED,
+  STATS_PASSED,
+  STATS_SKIPPED,
+} from 'common/constants/statistics';
+import { FAILED, PASSED, SKIPPED } from 'common/constants/launchStatuses';
+import { COLOR_FAILED, COLOR_PASSED, COLOR_SKIPPED } from 'common/constants/colors';
+import {
+  PRODUCT_BUG,
+  AUTOMATION_BUG,
+  SYSTEM_ISSUE,
+  NO_DEFECT,
+  TO_INVESTIGATE,
+} from 'common/constants/defectTypes';
+import { connect } from 'react-redux';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { withModal, ModalLayout } from 'components/main/modal';
+import { SpinningPreloader } from 'components/preloaders/spinningPreloader';
+import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+import { defectColorsSelector } from 'controllers/project';
+import { activeProjectSelector } from 'controllers/user';
+import styles from './launchCompareModal.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  compareLaunchHeader: {
+    id: 'CompareLaunchDialog.compareLaunchHeader',
+    defaultMessage: 'Compare launches',
+  },
+  yAxisTitle: {
+    id: 'CompareLaunchDialog.yAxisTitle',
+    defaultMessage: '% of test cases',
+  },
+  passed: {
+    id: 'LaunchStatus.passed',
+    defaultMessage: 'Passed',
+  },
+  failed: {
+    id: 'LaunchStatus.failed',
+    defaultMessage: 'Failed',
+  },
+  skipped: {
+    id: 'LaunchStatus.skipped',
+    defaultMessage: 'Skipped',
+  },
+  product_bug: {
+    id: 'DefectType.product_bug',
+    defaultMessage: 'Product bug',
+  },
+  automation_bug: {
+    id: 'DefectType.automation_bug',
+    defaultMessage: 'Automation bug',
+  },
+  system_issue: {
+    id: 'DefectType.system_issue',
+    defaultMessage: 'System issue',
+  },
+  no_defect: {
+    id: 'DefectType.no_defect',
+    defaultMessage: 'No defect',
+  },
+  to_investigate: {
+    id: 'DefectType.to_investigate',
+    defaultMessage: 'To investigate',
+  },
+});
+const chartConfig = {
```

Comment:
Since you modify `chartConfig` in the runtime, I think it's better to define a function which returns a new config, so there will be no information about previous usages.